app/util/participant_edit.py

```
@@ -1,55 +1,78 @@
     for member in members:
         try:
-            country_list.add(member["Country"]["names"][language])
+            if language:
+                country_list.add(member["Country"]["names"][language])
             iso_alpha2_list.add(member["Country"]["iso_alpha2"])
         except KeyError:
-            raise ValueError(
-                "ParticipantMemberにCountry(names, iso_alpha2)が存在しません Participantテーブルを取得する際に、Country(names, iso_alpha2)をjoinさせてください"
-            )
+            if language:
+                raise ValueError(ISO_CODE_COUNTRY_NAMES_OR_ALPHA2_NOT_FOUND)
```

Ruff (0.14.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)